pkg/apigee/envVar.go

```diff
@@ -0,0 +1,115 @@
+package apigee
```

Comment:
Personally i would have probably left the k8s stuff out of the apigee pkg incase at some point we wanted to pull that pkg into it's on Apigee golang sdk sort of thing.